github/ipld.yml

```
@@ -3105,48 +3030,77 @@ teams:
   Admin:
     members:
       maintainer:
-        - aschmahmann
-        - daviddias
-        - jbenet
-        - Stebalien
         - vmx
-        - warpfork
-        - whyrusleeping
-      member:
-        - BigLep
-        - Kubuxu
     privacy: closed
   Alumni:
-    privacy: closed
-  ci:
-    description: ci
     members:
       member:
+        - 0xDanomite
+        - aarshkshah1992
+        - AdamStone
+        - adlrocha
+        - ajnavarro
+        - andrew
+        - andyschwab-admin
+        - andyschwab
+        - anorth
+        - arajasek
+        - aschmahmann
+        - cloutiertyler
+        - davidad
+        - davidd8
+        - daviddias
+        - dhruvbaldawa
+        - dignifiedquire
+        - dirkmc
+        - flyingzumwalt
+        - guseggert
+        - haadcode
+        - hacdias
+        - hugomrdias
         - ipfsbot
-    privacy: closed
-  close-collaborators:
-    description: Folks who work across many IPLD repos and languages
-    members:
-      maintainer:
-        - warpfork
-      member:
+        - ipldbot
+        - jacobheun
+        - jbenet
+        - jbenetsafer
+        - jesseclay
+        - Jorropo
+        - juliaxbow
+        - kevina
+        - Kubuxu
+        - kumavis
+        - lidel
+        - litzenberger
+        - magik6k
+        - MarcoPolo
+        - marten-seemann
+        - mcollina
+        - michaelavila
+        - miyazono
+        - momack2
+        - mxinden
+        - nicola
+        - petar
+        - pkafei
         - RangerMauve
+        - raulk
+        - ribasushi
+        - RichardLitt
+        - richardschneider
+        - tchardin
+        - tinytb
+        - travisperson
+        - vasco-santos
+        - victorb
+        - wanderer
+        - warpfork
+        - whizzzkid
+        - whyrusleeping
     privacy: closed
   Contributors:
     members:
       maintainer:
-        - aschmahmann
-        - daviddias
-        - Stebalien
         - vmx
-        - whyrusleeping
-      member:
-        - adlrocha
-        - davidad
-        - flyingzumwalt
-        - Kubuxu
-        - nicola
-        - RichardLitt
     privacy: closed
   Core:
     description: Core Team
```

```suggestion
  General Maintainers:
    description: Invested maintainers who can act as a catch-basin in the absence of anyone else.
```

I think this name is more clear?  I don't think we want to imply that there is a "Core" team who are dedicated to IPLD, since that isn't the case currently.